src/visitpy/CMakeLists.txt

```diff
@@ -58,9 +77,9 @@
 #####################
 # Metadata Objects that we can't Code-Gen, or have customizations:
 #####################
-# avtMeshMetaData
-# avtSimulationInformation
-# avtSpeciesMetaData
+# avtMeshMetaData -- OK?
+# avtSimulationInformation -- OK?
+# avtSpeciesMetaData -- OK?
 
```

Comment:
checkout if non auto gen is a surpise here, did we port correct?

Reply:
I checked autogen of these on current develop, and I believe the autogen problems were resolved previously, so making sure they are part of autogen going forward: +1